modules/common/src/main/scala/com/snowplowanalytics/snowplow/postgres/streaming/sink.scala

```diff
@@ -41,59 +41,54 @@ object sink {
   /**
    * Sink good events into Postgres. During sinking, payloads go through all transformation steps
    * and checking the state of the DB itself.
-   * Events that could not be transformed (due Iglu errors or DB unavailability) are sent back
-   * to `badQueue`
+   * Events that could not be transformed (due Iglu errors or DB unavailability) are emitted from
+   * the pipe
```

Comment:
@chuwy can you comment on why a `Queue` was chosen in the first place?

I felt like the queue was introducing an extra piece of mutable state where it wasn't needed.  By removing the intermediate queue there is a more direct flow of events from end to end.

My biggest concern was that the queue was bounded, so events would be dropped if the queue fills up.  Changing it to unbounded would not make me feel better, because there are still questions about what happens if the queue gets more and more full.  But by streaming directly end to end, I think there is no opportunity for events to build up at a bottleneck.